app/controllers/api/v1/questions_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::QuestionsController < Api::BaseController
+  # For daily quiz question
+  def daily
+    render json: Question.active('daily').not_seen(@api_current_user.id).first
```

Comment:
Can we move the daily/weekly distinction to a different scope too ?
Like `Question.active.daily` instead of `Question.active('daily')`
Because here we are binding the category to the active/inactive clause, which I think might not be required.

Reply:
Done. Thanks